operations/agent-flow-mixin/dashboards/cluster-overview.libsonnet

```diff
@@ -0,0 +1,231 @@
+local dashboard = import './utils/dashboard.jsonnet';
+local panel = import './utils/panel.jsonnet';
+local filename = 'agent-cluster-overview.json';
+
+{
+  [filename]:
+    dashboard.new(name='Grafana Agent Flow / Cluster Overview') +
+    dashboard.withDocsLink(
+      url='https://grafana.com/docs/agent/latest/flow/reference/cli/run/#clustered-mode-experimental',
+      desc='Clustering documentation',
+    ) +
+    dashboard.withDashboardsLink() +
+    dashboard.withUID(std.md5(filename)) +
+    dashboard.withTemplateVariablesMixin([
+      dashboard.newTemplateVariable('cluster', |||
+        label_values(agent_component_controller_running_components, cluster)
+      |||),
+      dashboard.newTemplateVariable('namespace', |||
+        label_values(agent_component_controller_running_components{cluster="$cluster"}, namespace)
+      |||),
+    ]) +
+    dashboard.withPanelsMixin([
+      // Nodes
+      (
+        panel.new('Nodes', 'stat') +
+        panel.withDescription(|||
+          Node count.
+        |||) +
+        panel.withPosition({ h: 9, w: 8, x: 0, y: 0 }) +
+        panel.withQueries([
+          panel.newInstantQuery(
+            expr='count(cluster_node_info)'
+          ),
+        ])
+      ),
+      // Node table
+      (
+        panel.new('Node table', 'table') +
+        panel.withDescription(|||
+          Nodes info.
+        |||) +
+        panel.withPosition({ h: 18, w: 16, x: 8, y: 0 }) +
+        panel.withQueries([
+          panel.newInstantQuery(
+            expr='cluster_node_info',
+            format='table',
+          ),
+        ]) +
+        panel.withTransformations([
+          {
+            id: 'organize',
+            options: {
+              excludeByName: {
+                Time: true,
+                Value: false,
+                __name__: true,
+                cluster: true,
+                namespace: true,
+                state: false,
+              },
+              indexByName: {},
+              renameByName: {
+                Value: 'Dashboard',
+                instance: '',
+                state: '',
+              },
+            },
+          },
+        ]) +
+        panel.withOptions({
+          footer: {
+            countRows: false,
+            fields: '',
+            reducer: [
+              'sum',
+            ],
+            show: false,
+          },
+          showHeader: true,
+          sortBy: [
+            {
+              desc: true,
+              displayName: 'instance',
+            },
+          ],
+        }) +
+        panel.withFieldConfigs({
```

Comment:
Some of these field configs may not be necessary. Anything which is the default supplied by Grafana can be omitted, which will be helpful to make the libraries smaller. 

The best way of checking to see if something matters is to run `grr watch` and see if anything changes when modifying the file.

For example, in the case of the table panels, there's no reason to supply the thresholds for values, since there's no values to color green/red.